### PR TITLE
Add translation ko.yml for korean

### PR DESCRIPTION
**Pull request by yhk1038**
Thanks to make the awesome gem. 👍(duplicate commit inside a last one)

### CODE REVIEW

File: lib/locales/ko.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/lib/locales/utils/p11n.rb)
+
+ko:
+  pagy:
+
+    item_name:
+      one: "항목"
```
Comment (ddnexus) [inline lib/locales/ko.yml]: I think `:one` is not used in the `ko` pluralization (which should be `:other` type)

File: lib/locales/ko.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/lib/locales/utils/p11n.rb)
```
Comment (ddnexus) [inline lib/locales/ko.yml]: I think this pluralization should be `:other` and not `:one_other`

**Comment by ddnexus** [comment]
Hi @yhk1038, thank you for the PR.

According to the `ko` rails pluralization [file](https://github.com/svenfuchs/rails-i18n/blob/master/rails/pluralization/ko.rb) the pluralization to use should be `:other`.

File: lib/locales/ko.yml (hunk 2 above)
Comment (yhk1038) [reply lib/locales/ko.yml]: Oh, i just skipped this line means. I'll check my fault with the other comment below and PR again. Thank your review 👍

**Comment by ddnexus** [comment]
It looks good now. Thanks for your contribution!

**Pull request merged**